modules.d/01fips/fips.sh

```diff
@@ -81,6 +94,33 @@ do_rhevh_check() {
     return 0
 }
 
+do_uki_check() {
+    KVER=$(uname -r)
+    # UKI are placed in $ESP\EFI\Linux\<intall-tag>-<uname-r>.efi
+    if ! [ "$FIPS_MOUNTED_BOOT" = 1 ]; then
+        warn "Failed to mount ESP for doing UKI integrity check"
+        return 1
+    fi
+
+    uki_checked=0
+    for UKIpath in /boot/EFI/Linux/*-"$KVER".efi; do
+        [ -f "$UKIpath" ] || break
```

Comment:
What is this check for?

Maybe let's use `-r` to check it's readable (i.e. we have permissions to read it)?

Reply:
This is mostly for the case, when there are no UKIs matching '/boot/EFI/Linux/*-"$KVER".efi' and we actually get a string with '*' :-) Switching to '-r' is nevertheless a good idea, will do!